packages/transaction-controller/CHANGELOG.md

```diff
@@ -7,6 +7,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [33.1.0]
```

Comment:
It doesn't seem that there are any new features in this release. Should this be 33.0.1 rather than 33.1.0, following [SemVer](https://semver.org/)?

Reply:
Updated in [91a6881](https://github.com/MetaMask/core/pull/4460/commits/91a6881991b2079de6fa5f0e10446c434ddb17cd) and [2cf1e08](https://github.com/MetaMask/core/pull/4460/commits/2cf1e08601366c8262a2f8e0343140e77774a38d)